sympy/calculus/util.py

```diff
@@ -1223,19 +1223,11 @@ def __mul__(self, other):
     def __div__(self, other):
         if isinstance(other, Expr):
             if isinstance(other, AccumBounds):
-                if S.Zero not in other:
+                if ((other.min.is_extended_positive and other.max.is_positive) or (other.min.is_negative and other.max.is_negative)):
```

Comment:
```suggestion
               if other.min.is_positive or other.max.is_negative:
```
Something like this should work because min is less than or equal to max.